src/includes/content/wizard/affliate/flycricket.pug

```diff
@@ -1,15 +1,5 @@
-#fc-container.box
-    #fc-container-text
-        .fc-title Free privacy policy hosting + website for {{ appName }}.
-    #fc-callouts
-        form#fc-form(action='https://app.flycricket.com/import/privacy' method='post' target='_blank' onsubmit='return fc_submit(event)')
-            input(type='hidden' name='fc-referrer' value='nishant')
-            input#fc-terms(type='hidden' name='fc-terms')
-            input#fc-body(type='hidden' name='fc-privacy')
-            input#fc-name(type='hidden' name='fc-name' v-model='appName')
-        div(style='margin-top: 1em;')
-            button.button.is-black.is-small(@click="fc_deploy()")
-                | Deploy
-        .fc-powered-by(style='margin-top:0.25em;')
-            | Powered by 
-            a(href='https://app.flycricket.com/r/nishant?utm_source=partner&utm_medium=website&utm_campaign=privacy_generator' target='_blank' rel='noopener noreferrer') Flycricket
+form#fc-form(action='https://app.flycricket.com/import/privacy' method='post' target='_blank' onsubmit='return fc_submit(event)')
+    input(type='hidden' name='fc-referrer' value='nishant')
+    input#fc-terms(type='hidden' name='fc-terms')
+    input#fc-body(type='hidden' name='fc-privacy')
+    input#fc-name(type='hidden' name='fc-name' v-model='appName')
```

Comment:
After reading your other comments and thinking about this a little further, I removed the flycricket box from being visible, but left the form, which is still needed for sending data to flycricket.

In this update, the deploy can _only_ be triggered from the modal. My thought process is:

The two end actions for the user are 'copy' or 'deploy'. In the existing flow, these were on separate pages (step3 vs modal). In this commit, all actions for the user are in one place so the user can easily decide what to do with the generated policy/terms. Plus it cleans up your step3 to direct the users right to the generated content.

What do you think @nisrulz? I think this'll increase deployments, so I might be a little biased :) but I do think it's a smoother flow, especially since terms have been added to step3.

Reply:
No that makes sense. The user has a choice here. If they want to deploy they can right away. The UX feels more in line with how someone would use this vs having to close the modal and hitting deploy. I will validate this PR on local before merging in.